pgjdbc/src/main/java/org/postgresql/util/PGInterval.java

```diff
@@ -239,12 +242,18 @@ public void setValue(int years, int months, int days, int hours, int minutes, do
    * @return String represented interval
    */
   public String getValue() {
-    return years + " years "
-        + months + " mons "
-        + days + " days "
-        + hours + " hours "
-        + minutes + " mins "
-        + wholeSeconds + '.' + microSeconds + " secs";
+    return String.format(
+      Locale.ROOT,
+      "%d years %d mons %d days %d hours %d mins %s secs",
+      years,
+      months,
+      days,
+      hours,
+      minutes,
+      new DecimalFormat("0.0#####").format(
+        wholeSeconds + (double) microSeconds / TimeUnit.SECONDS.toMicros(1)
```

Comment:
@davecramer Don't know if there's a more clever way here but since I need a double and TimeUnit only works with longs, avoiding the hard coded coefficient while still doing the arithmetic manually was the only gain I could see.

Also started using DecimalFormat because there were some tests that required seconds to always include a decimal place. Not sure if that's a hard requirement or not.